scripts/build-windows.ps1

```diff
@@ -170,7 +170,7 @@ else
 Write-Host "*************** Building recipes ***************"
 # Note: 'recipes' is already using only pnpm - can switch to $BASE_CMD AFTER both repos are using pnpm
 Push-Location recipes
-pnpm i && pnpm lint && pnpm reformat-files && pnpm package
+pnpm i; pnpm lint; pnpm reformat-files; pnpm package
```

Comment:
Ahh this probably shouldn't have been tacked on this merge request but the script currently fails because & does something different in powershell than CMD or bash. Will remove from this PR